l10n_it_declaration_of_intent/models/account_move.py

```diff
@@ -81,7 +81,10 @@ def select_manually_declarations(self):
     def _post(self, soft=True):
         posted = super()._post(soft)
         # Check if there is enough available amount on declarations
-        for invoice in self:
+        for invoice in self.filtered(
+            lambda i: i.move_type
+            in ["out_invoice", "out_refund", "in_invoice", "in_refund"]
```

Comment:
```suggestion
            lambda i: i.is_invoice()
```
Sono da includere anche le ricevute?

Reply:
teoricamente per il modulo non dovrebbe avere problemi poiche "in_receipt" e "out_receipt" iniziano con "in" e "out", ma non so se contabilmente dovrebbero essere incluse le ricevute